cpp/modmesh/view/R3DWidget.hpp

```diff
@@ -39,27 +39,13 @@
 #include <Qt3DCore/QEntity>
 #include <Qt3DRender/QCamera>
 
-#include <QAbstractCameraController>
-#include <QOrbitCameraController>
-#include <QFirstPersonCameraController>
+#include <modmesh/view/RCameraController.hpp>
 
 #include <QResizeEvent>
 
 namespace modmesh
 {
 
-class RFirstPersonCameraController
-    : public Qt3DExtras::QFirstPersonCameraController
-{
-    using Qt3DExtras::QFirstPersonCameraController::QFirstPersonCameraController;
-}; /* end class RFirstPersonCameraController */
-
-class ROrbitCameraController
-    : public Qt3DExtras::QOrbitCameraController
-{
-    using Qt3DExtras::QOrbitCameraController::QOrbitCameraController;
-}; /* end class ROrbitCameraController */
-
```

Comment:
Key difference in this PR, is that we don't use Qt original camera controller function ([moveCamera](https://github.com/solvcon/modmesh/pull/385/files#diff-d22d8f512997b8a57a09a8179b7446c525a92a5797d54e5e8dfa0922555814bbR134)), but instead implement our own controller function ([updateCameraPosition](https://github.com/solvcon/modmesh/pull/385/files#diff-d22d8f512997b8a57a09a8179b7446c525a92a5797d54e5e8dfa0922555814bbR136)).